portalnet/src/overlay/command.rs

```diff
@@ -40,3 +42,17 @@ pub enum OverlayCommand<TContentKey> {
     /// Handle an event sent from another overlay.
     Event(EventEnvelope),
 }
+
+/// A configuration for a query to find content.
+#[derive(Debug, Default)]
+pub struct FindContentConfig {
```

Comment:
nit: I think we can move this struct to `/overlay/config.rs` but I don't have very strong opinion.